CraftTweaker2-API/src/main/java/crafttweaker/api/util/IAxisAlignedBB.java

```diff
@@ -0,0 +1,62 @@
+package crafttweaker.api.util;
+
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.world.IVector3d;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenGetter;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+@ZenClass("crafttweaker.util.IAxisAlignedBB")
+@ZenRegister
+public interface IAxisAlignedBB {
```

Comment:
I'm not sure about adding an AABB Wrapper to CrT, since there's already a wrapper for it on the ContentTweaker side which might lead to confusion.

Do you already use that somewhere?